src/follower.py

```diff
@@ -73,21 +77,63 @@ def stop(self):
 
 
     """
-    Turn the wheels towards the last recognized object.
+    Turn the wheels and pans the camera towards the last recognized object.
     """
     def turn(self):
         self.convert_camera_angle()
-        self._fw.turn(self._turn_angle)
+        self.pan_camera()
+
+        if self._turn_angle > 90 and self._turn_angle < self.wheel_max:
+            self._fw.turn(self._turn_angle + self.camera_angle_offset) 
+        elif self._turn_angle > self.wheel_max:
+            self._turn_angle = self.wheel_max
+            self._fw.turn(self._turn_angle)
+        elif self._turn_angle < 90 and self._turn_angle > self.wheel_min:
+            self._fw.turn(self._turn_angle - self.camera_angle_offset)
+        elif self._turn_angle < self.wheel_min:
+            self._turn_angle = self.wheel_min
+            self._fw.turn(self._turn_angle)
+        
+    
+
+    """
+    Follows the tag by panning camera towards the same direction as the wheels.
+    """
+    def pan_camera(self):
+        if self._turn_angle < 90:
+            self.turn_camera_left(self._turn_angle)
+            self.camera_angle_offset = (90 - self._turn_angle)
```

Comment:
There's no scale conversion here. They're not on the same scale so you can't just subtract and call it good.

Reply:
created scale conversion function

Reply:
Where?

Reply:
its in the newest commit